code/__DEFINES/tablecrafting.dm

```diff
@@ -0,0 +1,7 @@
+
+#define CAT_NONE	""
+#define CAT_WEAPON	"Weaponry"
+#define CAT_AMMO	"Ammunition"
+#define CAT_ROBOT	"Robots"
+#define CAT_FOOD	"Food"
```

Comment:
Because magic strings are always defines so that you don't fuck up by spelling it wrong.